CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 2.11.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.3.1391** 
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented support of ``firewall_image_version`` parameter into **aviatrix_firewall_instance**
+2. Implemented support of ``vpn_protocol`` for vpn gateway of aws provider with elb enabled in **aviatrix_gateway**
+3. Implemented support of ``enable_active_active`` parameter in **aviatrix_site2cloud**
+
+### Enhancements:
+1. Implemented coverage for ``tag_list`` formatting change due to Boto3
+2. Implemented support of multiple security domains in **aviatrix_aws_tgw_vpn_conn**
```

Comment:
2. Implemented support for attaching TGW VPN connections to different security domains besides the default domain